test/integration/scheduler/scheduler_test.go

```diff
@@ -140,7 +140,7 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 	schedPredicates := sched.Config().Algorithm.Predicates()
 	schedPrioritizers := sched.Config().Algorithm.Prioritizers()
 	// Includes one mandatory predicates.
-	if len(schedPredicates) != 3 || len(schedPrioritizers) != 2 {
+	if len(schedPredicates) != 7 || len(schedPrioritizers) != 2 {
```

Comment:
Comment needs to be updated. Also would you make it more obvious where 7 and 2 come from?